sdk/formrecognizer/azure-ai-formrecognizer/tests/test_receipt.py

```diff
@@ -421,3 +421,30 @@ def test_receipt_continuation_token(self, client):
         result = poller.result()
         self.assertIsNotNone(result)
         initial_poller.wait()  # necessary so azure-devtools doesn't throw assertion error
+
+    @GlobalFormRecognizerAccountPreparer()
+    @GlobalClientPreparer()
+    def test_receipt_locale_specified(self, client):
+        with open(self.receipt_jpg, "rb") as fd:
+            receipt = fd.read()
+        poller = client.begin_recognize_receipts(receipt, locale="en-IN")
+        assert 'en-IN' == poller._polling_method._initial_response.http_response.request.query['locale']
+        poller.wait()
+
+    @GlobalFormRecognizerAccountPreparer()
+    @GlobalClientPreparer()
+    def test_receipt_locale_error(self, client):
+        with open(self.receipt_jpg, "rb") as fd:
+            receipt = fd.read()
+        with pytest.raises(HttpResponseError) as e:
+            client.begin_recognize_receipts(receipt, locale="not a locale")
+        assert "locale" in e.value.error.message
```

Comment:
sorry didn't notice this before, let's not assert on error messages since they can change. Asserting on error code should be fine.